safe/gui/tools/test/test_options_dialog.py

```diff
@@ -26,45 +26,106 @@
 class TestOptionsDialog(unittest.TestCase):
     """Test Options Dialog"""
 
+    def setUp(self):
+        """Fixture run before all tests"""
+        self.qsetting = QSettings('InaSAFETest')
+        self.qsetting.clear()
+
+    def tearDown(self):
+        """Fixture run after each test"""
+        # Make sure it's empty
+        self.qsetting.clear()
+
     def test_setup_dialog(self):
         """Test Setup Options Dialog."""
-        dialog = OptionsDialog(PARENT, IFACE, qsetting='InaSAFETest')
+        dialog = OptionsDialog(
+            parent=PARENT, iface=IFACE, qsetting='InaSAFETest')
         self.assertIsNotNone(dialog)
 
         # Check default values
-        self.assertTrue(dialog.cbxVisibleLayersOnly.isChecked())
-        self.assertTrue(dialog.cbxSetLayerNameFromTitle.isChecked())
-        self.assertTrue(dialog.cbxZoomToImpact.isChecked())
-        self.assertFalse(dialog.cbxHideExposure.isChecked())
-        self.assertFalse(dialog.cbxUseSelectedFeaturesOnly.isChecked())
+        self.assertEqual(
+            dialog.cbxVisibleLayersOnly.isChecked(),
+            inasafe_default_settings['visibleLayersOnlyFlag'])
+        self.assertEqual(
+            dialog.cbxSetLayerNameFromTitle.isChecked(),
+            inasafe_default_settings['set_layer_from_title_flag'])
+        self.assertEqual(
+            dialog.cbxZoomToImpact.isChecked(),
+            inasafe_default_settings['setZoomToImpactFlag'])
+        self.assertEqual(
+            dialog.cbxHideExposure.isChecked(),
+            inasafe_default_settings['setHideExposureFlag'])
+        self.assertEqual(
+            dialog.cbxUseSelectedFeaturesOnly.isChecked(),
+            inasafe_default_settings['useSelectedFeaturesOnly'])
         self.assertEqual(
             dialog.leKeywordCachePath.text(),
-            dialog.keyword_io.default_keyword_db_path())
-        self.assertTrue(dialog.template_warning_checkbox.isChecked())
+            inasafe_default_settings['keywordCachePath'])
+        self.assertEqual(
+            dialog.template_warning_checkbox.isChecked(),
+            inasafe_default_settings['template_warning_verbose'])
+        self.assertEqual(
+            dialog.organisation_on_dock_checkbox.isChecked(),
+            inasafe_default_settings['showOrganisationLogoInDockFlag'])
+        self.assertEqual(
+            dialog.cbxDevMode.isChecked(),
+            inasafe_default_settings['developer_mode'])
```

Comment:
Same with this, I'd rather have for loops if its doing the same thing and have the assert displays the relevant key if it's not equal.